AcadeMeData/tests.py

```diff
@@ -1,3 +1,34 @@
-# from django.test import TestCase
-
-# Create your tests here.
+from django.test import TestCase
+from AcadeMeData.models import Users
+
+
+class TestUserModel(TestCase):
+
+    def test_create_user(self):
+        username = "username"
+        password = "password"
+        email = "user@example.com"
+        type = "S"
+        university = "RU"
+        degree = "CS"
+        user = Users.create_user(
+            username, email, password, type, university, degree)
+        assert isinstance(user, Users)
+        assert user.user.username == username
+        assert user.user.password == password
+        assert user.user.email == email
+        assert user.user.type == type
+        assert user.user.university == university
+        assert user.user.degree == degree

```

Comment:
A model should, in most cases, represent a single entity.
So switch "class Users" with "class User" and make sure that the tests look like this:

assert user.username == username
assert user.password == password

Reply:
changes Applied

Reply:
+1

Reply:
RESOLVED